### PR TITLE
c2cupgrade script: typos in texts

### PR DESCRIPTION
**Pull request by jwkaltz**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py [hunk 1]
```diff
@@ -702,10 +703,10 @@ def step13(self, _):
         print("")
         print(self.color_bar)
         print("")
-        print(colorize("Congratulations your upgrade is a success.", GREEN))
+        print(colorize("Congratulations, your upgrade was successful.", GREEN))
         print("")
         branch = check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]).decode("utf-8").strip()
-        print("Now all your files will be committed, you should do a git push:")
+        print("Now all your files can be committed, you should do a git push:")
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: is commited

File: geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py [hunk 2]
```diff
@@ -52,11 +52,12 @@
...
 }
-DIFF_NOTICE = "The changes visible on `a/CONST_create_template/<file>` should be done on `<file>`.\n" \
-    "An advise to be more effective: in most cases it concerns a file that you never customize, " \
-    "or a file that you have heavily customized, then respectively copy the new file from " \
-    "`CONST_create_template` (`cp CONST_create_template/<file> <file>`), respectively ignore " \
-    "the changes."
+DIFF_NOTICE = "You should apply the changes shown in the diff file on `a/CONST_create_template/<file>` " \
+    "on your project's `<file>`.\n" \
+    "Some advice to be more efficient: if the changes on a file concern a file that you never customize, " \
+    "you can simply copy the new file from `CONST_create_template` (`cp CONST_create_template/<file> <file>`)." \
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: , and ad it in the `unmanaged_files` section of the `project.yaml.mako` file.

File: geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py (hunk 1 above)
Comment (jwkaltz) [reply geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: OK, changing to "are committed"

File: geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py (hunk 2 above)
Comment (jwkaltz) [reply geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: I now added that, can you please check if I understood correctly?

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: You steal needs to copy them, the changes is for the next upgrade...

**Pull request merged**